packages/react-native-payments/src/@standard/android/mapping/android-payment-method-data-data.interface.ts

```diff
@@ -1,11 +1,11 @@
-import type { EnvironmentEnum } from '../../../enum/environment.enum';
 import type { GenericPaymentMethodDataDataInterface } from '../../../interface/generic-payment-method-data-data.interface';
+import type { AndroidAllowedAuthMethodsEnum } from '../enum/android-allowed-auth-methods.enum';
 import type { AndroidTokenizationDirectSpecification } from '../request/android-tokenization-direct-specification';
 import type { AndroidTokenizationGatewaySpecification } from '../request/android-tokenization-gateway-specification';
 
 interface AndroidGenericPaymentMethodDataInterface extends GenericPaymentMethodDataDataInterface {
-    // Android environment https://developers.google.com/android/reference/com/google/android/gms/wallet/Wallet.WalletOptions.Builder#setEnvironment(int)
-    environment: EnvironmentEnum;
+    // PAN_ONLY and CRYPTOGRAM_3DS by default
+    allowedAuthMethods?: AndroidAllowedAuthMethodsEnum[];
```

Comment:
you can find this configuration here `defaultAndroidPaymentDataRequest` it has setup for a 
```
allowedAuthMethods: [AndroidAllowedAuthMethodsEnum.PAN_ONLY, AndroidAllowedAuthMethodsEnum.CRYPTOGRAM_3DS],
``` 
by default, are you planning to pass other options? According to the [Android docs](https://developers.google.com/pay/api/android/reference/request-objects#PaymentMethod) we should only use PAN_ONLY and CRYPTOGRAM_3DS and they are already a default option, that is why I have considered not exposing it into the mapper. Can you please describe your use case?

Reply:
Yes, for our use case we want to have PAN_ONLY auth method, so I added the ability to override the auth methods. I don't see where it says in [Android docs](https://developers.google.com/pay/api/android/reference/request-objects#CardParameters) that we should only use both.

Reply:
Absolutely, then I suggest this changes,

and option to android config:
`packages/react-native-payments/src/@standard/android/mapping/android-payment-method-data-data.interface.ts`
```ts
interface AndroidGenericPaymentMethodDataInterface extends GenericPaymentMethodDataDataInterface {
    allowedAuthMethods?: AndroidAllowedAuthMethodsEnum[];
    // Android environment https://developers.google.com/android/reference/com/google/android/gms/wallet/Wallet.WalletOptions.Builder#setEnvironment(int)
    environment: EnvironmentEnum;
}
```


add mapping for this field: 
`packages/react-native-payments/src/class/payment-request/payment-request.ts`
```ts
allowedAuthMethods:
                            methodData.allowedAuthMethods ?? defaultAndroidPaymentMethod.parameters.allowedAuthMethods,
```

Reply:
yep, did it - https://github.com/rnw-community/rnw-community/pull/146/commits/1571f6749d4a865fd305007a1c3759380d9a3c25

Reply:
reverted